lib/cucumber/cli.js

```diff
@@ -51,7 +50,12 @@ function Cli(argv) {
 
   var self = {
     run: function run(callback) {
-      var configuration = getConfiguration();
+      var program = getProgram();
+      program.parse(argv);
```

Comment:
Lets not add an additional parse. We can add a method to Configuration to check if the init option was passed.